pootle/static/js/editor/app.js

```diff
@@ -2312,4 +2338,40 @@ PTL.editor = {
     return true;
   },
 
+  toggleSuggestion(e, canHide = false) {
+    e.stopPropagation();
+    const $suggestion = $(e.target).parents('.js-user-suggestion');
+
+    if (this.selectedSuggestionId === undefined) {
+      this.selectedSuggestionId = $suggestion.data('sugg-id');
+      const props = {
+        editor: this,
+        suggId: this.selectedSuggestionId,
+        suggText: $suggestion.data('translation-aid'),
+      };
+      const $feedbackPlaceHolder = $suggestion.find('.feedback')[0];
```

Comment:
This is a mount point, and as I see it's guaranteed to be only one at a time, so there doesn't seem to be the need to find it relative to another node. Also make sure to prefix such selectors with `js-`.

```js
const feedbackMountPoint = document.querySelector('.js-mnt-suggestion-feedback');
```